README.md

```diff
@@ -24,6 +21,23 @@ https://github.com/fridayshoes/acebook-node-priceless
 
 https://trello.com/b/jaHA0ziM/priceless-ace-book
 
+## Feature List
+
+- User can signup for an account
+   - Signing up twice with the same email address is not allowed
+- User can signin with an account
+- User can create a post
+   - A post can have multiple lines, it is displayed correctly on wall
+- User can delete a post
+- Users can react to a post by 
+   - liking a post (each user can only like a post once)
+   - commenting on a post
+- User has a Profile Page
+   - User can make posts to another user on Profile Page
+   - User can see posts made by themselves and made to them on Profile Page
+- User passwords are encrypted before they are stored in the database
+
```

Comment:
readme looks good

Reply:
- Dates are added to posts
- Posts are sorted with newest post first

Reply:
@fridayshoes Added both.